src/index.js

```diff
@@ -0,0 +1,52 @@
+import './style.css';
+
+const listItems = document.querySelector('.task-list');
+const addTask = document.querySelector('#new-task');
+const addBtn = document.getElementById('plus');
+
+let taskArr = [];
+
+const addNewList = () => {
+  const task = {};
+  task.index = taskArr.length + 1;
+  task.description = addTask.value;
+  task.completed = false;
+  taskArr.push(task);
+};
+const pushToLocal = () => {
+  localStorage.setItem('taskArr', JSON.stringify(taskArr));
+};
+
```

Comment:
- [ ] you are required in this milestone to populate the task array yourself, adding tasks using the form is a requirement for the next milestone. please make sure to follow the instructions.